### PR TITLE
Blobfuse flexvolume

### PR DESCRIPTION
**Pull request by ingeknudsen**
Issue:
https://github.com/equinor/radix-platform/issues/372

### CODE REVIEW

File: pkg/apis/deployment/volumemount.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+		namespace := deploy.radixDeployment.Namespace
+
+		for _, volumeMount := range deployComponent.VolumeMounts {
+			if volumeMount.Type == radixv1.MountTypeBlob {
+				secretName := defaults.GetBlobFuseCredsSecret(deployComponent.Name)
+
+				flexVolumeOptions := make(map[string]string)
+				flexVolumeOptions["container"] = volumeMount.Container
+				flexVolumeOptions["mountoptions"] = defaultMountOptions
+				flexVolumeOptions["tmppath"] = fmt.Sprintf(tempPath, namespace, deployComponent.Name)
```
Comment (nilsgstrabo) [inline pkg/apis/deployment/volumemount.go:52]: Use folder structure instead `/tmp/<app>/<omp name>/<env>`

**Review by nilsgstrabo — CHANGES_REQUESTED** [review]
Fix tmp folder mounting name

File: pkg/apis/deployment/volumemount.go (hunk 1 above)
Comment (satr) [reply pkg/apis/deployment/volumemount.go]: Fixed

**Review by nilsgstrabo — APPROVED** [review]
Need to implement validation of environment and component name to prevent path injection

**Pull request merged**